node/pkg/devnet/deterministic_bridge_key_test.go

```diff
@@ -0,0 +1,32 @@
+package devnet
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDeterministicEcdsaKeyByIndex(t *testing.T) {
+	type test struct {
+		index      uint64
+		privKeyHex string
+	}
+
+	tests := []test{
+		{index: 0, privKeyHex: "cfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2ee113a0"},
+		{index: 1, privKeyHex: "c3b2e45c422a1602333a64078aeb42637370b0f48fe385f9cfa6ad54a8e0c47e"},
+		{index: 2, privKeyHex: "9f790d3f08bc4b5cd910d4278f3deb406e57bb5e924906ccd52052bb078ccd47"},
+		{index: 3, privKeyHex: "b20cc49d6f2c82a5e6519015fc18aa3e562867f85f872c58f1277cfbd2a0c8e4"},
+	}
```

Comment:
Since you're defining the struct and then an instance immediately after, you can just use an anonymous struct. This is shorter and also valid go:

```suggestion
	tests := []struct{
		index      uint64
		privKeyHex string
	}{
		{index: 0, privKeyHex: "cfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2ee113a0"},
		{index: 1, privKeyHex: "c3b2e45c422a1602333a64078aeb42637370b0f48fe385f9cfa6ad54a8e0c47e"},
		{index: 2, privKeyHex: "9f790d3f08bc4b5cd910d4278f3deb406e57bb5e924906ccd52052bb078ccd47"},
		{index: 3, privKeyHex: "b20cc49d6f2c82a5e6519015fc18aa3e562867f85f872c58f1277cfbd2a0c8e4"},
	}
```

Might be a little bit better to do it this way? It is generally how I do these for unit tests personally. Here is [an example](https://git.w2k.jumptrading.com/linux/automaton/pull/770/files#diff-81a6e4b8db3a69352980fed1e2b64ca2d5be502705f98b76cafd18cb40d0073aR2149) of code I wrote doing it this way some time ago.

Reply:
This is good feedback, I'm going to include it on future cases.